doc/visual-programming/source/widgets/data/file.rst

```diff
@@ -16,32 +16,31 @@ Signals
 
 -  **Data**
 
-Attribute-valued data from the input file.
+Attribute-valued data from the input file
 
 Description
 -----------
 
-**File** widget :doc:`reads the input data
+The **File** widget :doc:`reads the input data
 file <../../loading-your-data/index>` (data table
 with data instances) and sends the data set to its output channel.
-History of the most recently opened files is maintained in the widget.
+The history of most recently opened files is maintained in the widget.
 The widget also includes a directory with sample data sets that come
 pre-installed with Orange.
 
 The widget reads data from Excel (**.xlsx**), simple tab-delimited
-(**.txt**) or comma-separated files (**.csv**).
+(**.txt**), comma-separated files (**.csv**) or URLs. 
 
 .. figure:: images/File-stamped.png
 
-1. Browse for a data file.
-2. Browse through previously opened data files, or load any of the
-   sample ones.
+1. Browse through previously opened data files, or load any of the sample ones.  
+2. Browse for a data file.
 3. Reloads currently selected data file.
-4. Information on the loaded data set: data set size, number and types
-   of data features.
-5. Allows you to distinguish between columns with the same name across
-   files (otherwise columns with the same name will be considered as the
-   same attribute).
+4. Insert data from a URL. 
```

Comment:
It also accepts data files from URL addresses. So, can add ", including data from Google Sheets"